atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/lock/EteLockWatchResource.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.lock;
+
+import com.palantir.lock.watch.CommitUpdate;
+import com.palantir.lock.watch.TransactionsLockWatchUpdate;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Path("lock-watch")
+public interface EteLockWatchResource {
+    @POST
+    @Path("start-transaction")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
I know I don't need so many annotations - will remove when I want to get this in.